release_images.yml

```diff
@@ -2,182 +2,45 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "huggingface_pytorch"
-    version: "1.10.2"
-    arch_type: "x86"
-    hf_transformers: "4.17.0"
-    training:
-      device_types: [ "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
-    inference:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
+    framework: "tensorflow"
+      version: "2.7.0"
```

Comment:
Looks like the indentation is not correct here. Can you please validate and indent the YML using https://onlineyamltools.com/validate-yaml or any other tool

Reply:
Resolved!